NAMESPACE

```diff
@@ -8,5 +8,17 @@ export(run_app)
 import(shiny)
 importFrom(COINr,get_corr_flags)
 importFrom(COINr,get_stats)
+importFrom(COINr,icodes_to_inames)
 importFrom(COINr,is.coin)
+importFrom(golem,with_golem_options)
+importFrom(shiny,HTML)
+importFrom(shiny,column)
+importFrom(shiny,shinyApp)
+importFrom(shiny,tagAppendAttributes)
+importFrom(shiny,tagList)
+importFrom(shiny,tags)
+importFrom(shinydashboard,menuItem)
+importFrom(shinydashboard,menuSubItem)
+importFrom(shinydashboard,tabItem)
```

Comment:
We don't really need all these because I am specifying e.g. `shinydashboard::menuItem` etc.